source/MapPanel.cpp

```diff
@@ -1152,17 +1153,37 @@ void MapPanel::DrawMissions()
 	const Color &blockedColor = *colors.Get("blocked mission");
 	const Color &specialColor = *colors.Get("special mission");
 	const Color &waypointColor = *colors.Get("waypoint");
+	if(specialSystem)
+	{
+		// The special system pointer is larger than the others.
+		pointerCount[specialSystem] += 1;
+		Angle a = Angle(30.*pointerCount[specialSystem] );
+		Point pos = Zoom() * (specialSystem->Position() + center);
+		PointerShader::Draw(pos, a.Unit(), 20.f, 27.f, -4.f, black);
+		PointerShader::Draw(pos, a.Unit(), 11.5f, 21.5f, -6.f, specialColor);
+	}
+	// Calculate the available (and unavailable) jobs, but don't draw them yet.
 	for(const Mission &mission : player.AvailableJobs())
 	{
 		const System *system = mission.Destination()->GetSystem();
-		DrawPointer(system, angle[system], mission.CanAccept(player) ? availableColor : unavailableColor);
+		if(mission.CanAccept(player))
+			++(availableCount[system].first);
```

Comment:
```suggestion
			++availableCount[system].first;
```

? There seems to be examples with and without throughout the code. We should really decide which one. My vote is without.